source/module.ts

```diff
@@ -17,11 +18,13 @@ export function formStoreFactory(ngRedux: NgRedux<any>) {
     ReactiveFormsModule,
   ],
   declarations: [
-    Connect,
+    ConnectTemplateDirective,
+    ConnectReactiveDirective,
     ConnectArray,
   ],
   exports: [
-    Connect,
+    ConnectTemplateDirective,
+    ConnectReactiveDirective,
```

Comment:
No need for a breaking change.
* `Connect` and
* `ReactiveConnect`
are fine, too. 

In `@angular/forms` its similar:
* `FormsModule`
* `ReactiveFormsModule `